### DISCUSSION TITLE
Nominate Michael Dawson to a Stacks maintainer

### DISCUSSION THREAD

**Discussion by mhdawson** [paketo-buildpacks/feedback#243] [Stacks Team]
After discussion in the last Paketo weekly meeting I'd like to self nominate as a stacks maintainer.  I've done the work to enable the [ubi-base-stack](https://github.com/paketo-community/ubi-base-stack) and [builder-ubi-buildpackless-base](https://github.com/paketo-community/builder-ubi-buildpackless-base) stacks and would like to help ensure that PRs are landed, releases are made etc.  

I'm happy to limit my involvement to the ubi stacks (which will likely have have most of my attention anyway), but also happy to try to help along PRs, releases, etc. for the other stacks as well if that makes sense.

**Comment by sophiewigmore** [comment]
@paketo-buildpacks/stacks (@brayanhenao please vote on this when you get a chance please!)

**Comment by sophiewigmore** [comment]
Sorry @mhdawson, our other two stacks maintainers are both of leave right now, so we've been blocked on getting 2/3 votes needed. I wonder if @paketo-buildpacks/steering-committee can help us get this through in their absence??

**Comment by loewenstein** [comment]
I don't know if we have established any rules for such situations and I kind of bet we haven't. On the other hand side, for the paketo-community organisation we anyway don't have too strict rules in place, have we?

I don't know what period of absence we are talking about, but maybe just unblocking @mhdawson with write permissions on UBI related repositories in the community org is enough to bridge the gap until stacks maintainers have quorum again?

@paketo-buildpacks/steering-committee what do you think? I would also not block steps beyond, just be careful to not set a wrong precedent. On the other hand side, a Paketo team not reaching quorum for an extended period of time sounds dangerous and we might actively want to establish rules for this.

**Reply by sophiewigmore** [reply]
@mhdawson would the plan Jan laid out suffice for now?

**Reply by mhdawson** [reply]
Write permissions on UBI related repositories in the community org works for me as a bridge.  How long are people away out of interest?

**Reply by dmikusa** [reply]
Governance states that Steering Committee members can vote in the absence of maintainers. I would say that applies here. If the other Steering Committee members agree, then I think we can just move ahead and vote on this. I've added my vote above.

> In extenuating circumstances, i.e. long term absence of a maintainer, steering committee members may act as maintainers of any team in a voting capacity.

I'm not exactly sure how the vote calculation works out in this situation, if it's 66% of stack maintainers + steering committee members (i.e. 66% of 6 = 4) or if it's just 66% of stack maintainers (66% of 3 = 2). If @loewenstein and @ForestEckhardt both vote and approve, it's a non-issue though, so maybe it's not that important.

**Reply by loewenstein** [reply]
Added my vote.

**Reply by sophiewigmore** [reply]
Thank you both @loewenstein @dmikusa ! Forest is out sick but I'll ping him about this when he's back. 
@mhdawson one is out until mid-Feb, and the other until the end of April

**Comment by mhdawson** [comment]
@sophiewigmore wondering what the next step is ?

**Reply by sophiewigmore** [reply]
@ForestEckhardt bump on this!

**Comment by sophiewigmore** [comment]
Looks like we have all the votes we need, so @mhdawson can be added to the stacks maintainer team! @paketo-buildpacks/steering-committee if you can add him, thanks for your help everyone. Congrats Michael!

**Reply by sophiewigmore** [reply]
He has been added

**Comment by mhdawson** [comment]
Many thanks!
